app/example.graphql

```diff
@@ -1,6 +1,6 @@
 schema
-  @server(port: 4000, enableGraphiql: true, enableQueryValidation: false, hostname: "0.0.0.0")
-  @upstream(baseURL: "http://jsonplaceholder.typicode.com", enableHttpCache: true) {
+  @server(port: 4000)
```

Comment:
I think we can drop port, as default port is already set to 8000. We will need graphiql otherwise the webpage won't work.